Doc/whatsnew/3.7.rst

```diff
@@ -366,6 +366,20 @@ Changes in the Python API
   (Contributed by Serhiy Storchaka in :issue:`19930`.)
 
 
+Changes in the C API
+--------------------
+
+* Function :c:func:`PySlice_GetIndicesEx` is considered not safe for
+  resizable sequences.  It takes the current length of the sequence, but
+  if the slice indices are not instances of :class:`int`, but objects that
```

Comment:
"[...] instances of :class:`int`" seems like an unended thoought. The double but in a row ("but if ..", "but objects") is quite confusing. Consider splitting into 2 or more sentences?